virttest/env_process.py

```diff
@@ -1047,6 +1050,14 @@ def preprocess(test, params, env):
                     env["cpu_driver"] = cpu_driver[0]
                     params["cpu_driver"] = env.get("cpu_driver")
 
+    kvm_module_params = params.get("kvm_module_parameters", "DEFAULT")
+    should_modify_kvm_module_params = kvm_module_params != "DEFAULT"
+    if should_modify_kvm_module_params:
+        global _kvm_module_handler
+        _kvm_module_handler = KernelModuleHandler('kvm')
+        kvm_module_force = params.get("kvm_module_force_load", "no") == "yes"
```

Comment:
Hmm, so in case of `DEFAULT` the `kvm_module_force_load` is not taken into consideration... that might be tricky... The more I think about this I think it'd be best to simply always call the KernelModuleHander and leave it to decide whether to do anything (even if just to load the kvm module)

Reply:
I mean imagine following scenario. I have `DEFAULT` set as well as `kvm_module_force_load` and I assume it will take care of loading the `kvm` module as it does when I have at least some configuration specified. Why would be `DEFAULT` different? It should still make sure the module is loaded even if it just used the default params to load it.

Reply:
> Hmm, so in case of `DEFAULT` the `kvm_module_force_load` is not taken into consideration... that might be tricky... The more I think about this I think it'd be best to simply always call the KernelModuleHander and leave it to decide whether to do 
anything (even if just to load the kvm module)

My take is that `KernelModuleHandler` should not be instantiated if it's not going to be used. But I understand your point. However, the constructor of `KernelModuleHandler` will currently run a process to read current configuration from the filesystem in order to back it up for later usage. I prefer this not to happen _always_. So, the backup logic would have to be moved out of the constructor and maybe into the `load_module` method. However, there's another solution (s. my next comment).

Reply:
> I mean imagine following scenario. I have `DEFAULT` set as well as `kvm_module_force_load` and I assume it will take care of loading the `kvm` module as it does when I have at least some configuration specified. Why would be `DEFAULT` different? It should still make sure the module is loaded even if it just used the default params to load it.

I didn't think of this use case: IIUC you mean the situation where I want the test to load a module without any parameters, e.g. a module that's normally not loaded per default on my system like `modprobe scsi_debug`. This would be possible by still using `DEFAULT` and setting `kvm_module_paramters = ""` and `kvm_module_force_load = "yes"`.

However, I prefer your proposal not to use `DEFAULT` and `""` instead (s. your comment further above).

Therefore, I propose we solve this with the following code:
```python
    kvm_module_params = params.get("kvm_module_parameters", "")
    should_modify_kvm_module_params = kvm_module_params != ""
    should_force_kvm_module_load = params.get("kvm_module_force_load", "no") == "yes"
    if should_modify_kvm_module_params or should_force_kvm_module_load:
        global _kvm_module_handler
        _kvm_module_handler = KernelModuleHandler('kvm')
        _kvm_module_handler.load_module(force=should_force_kvm_module_load, params=kvm_module_params)
```

Reply:
Thank you, I removed usage of `DEFAULT` and reviewed the design of argument usage. You can find the current behavior specified for `restore` and `load_module` in their respective doc strings.

Reply:
> Hmm, so in case of `DEFAULT` the `kvm_module_force_load` is not taken into consideration... that might be tricky... The more I think about this I think it'd be best to simply always call the KernelModuleHander and leave it to decide whether to do anything (even if just to load the kvm module)

Thank you. I added a module-level function that takes care of calling the load or not, please check.